Vagrantfile

```diff
@@ -51,17 +51,30 @@ end
 
 Vagrant.configure("2") do |config|
   (1..instances).each do |i|
-    config.vm.define "keylime#{i}" do |keylime|
-      keylime.vm.box = "fedora/33-cloud-base"
+    config.vm.define "keylime-fedora#{i}" do |keylime|
+      keylime.vm.box = "fedora/35-cloud-base"
       # Should you require machines to share a private network
       # Note, you will need to create the network first within
       # your provider (VirtualBox / Libvirt etc)
       # keylime.vm.network :private_network, ip: "10.0.0.#{i}1"
+
+      # Uncomment the following to forward ports on the VM and
+      # allow access to Keylime components from the host machine.
       keylime.vm.network "forwarded_port", guest: 443, host: "844#{i}"
```

Comment:
The forwarded ports are a great idea to make the test env more useful when you have other machines to connect. Thanks for that